TypeApprovalRegister.yaml

```diff
@@ -1384,28 +1384,28 @@ paths:
                     - 'RegistryOffice'
```

Comment:
subscriber-application :: /v1/redirect-info-about-approval-status-changes:
- The enum value "RegistryOffice" of  makes it more static. 
- Please let me know if we can remove this enum property and "== RegistryOffice" in last part of description

Reply:
Removed the enum and  "== RegistryOffice" .
Changed as proposed